ext-resources/ui.tcl

```diff
@@ -0,0 +1,235 @@
+#############################################################################
```

Comment:
What's this class? Also what's ui.tre, ui_support.py and ui.py (I'm assuming it's ui stuff but is it all necessary and does it work)?